### ISSUE TITLE
Pressing 'Enter' after add an Image Element doesn't enter in Edit Mode

### ISSUE THREAD

**Issue by obetomuniz** [GoogleForCreators/web-stories-wp#1830]
## Description

Images added on the workspace canvas should be able to enter in edit mode as soon as the user press <kbd>Enter</kbd> while having an image selected. Currently, images only enter edit mode if the user _**clicks**_ twice.

## Expected Behavior

Scenario 1:
1. Add an image from Media Library to the Workspace
2. Press <kbd>Enter</kbd>
3. Enter edit mode for images.

Scenario 2:
1. Select/focus an image in the workspace
2. Press <kbd>Enter</kbd>
3. Enter edit mode

**Comment by csossi** [comment]
Verified in QA - Tested PR#2753
